Explorer/Controller/ViewController.swift

```diff
@@ -31,8 +31,8 @@ class ViewController: UIViewController {
         view.addSubview(logTableView)
         installConstraints()
         generateDiveLogs()
-        let diveLogsView = viewModelsFrom(dataModels: diveLogs)
-        print(diveLogsView)
+        _ = viewModelsFrom(dataModels: diveLogs)
```

Comment:
Let's capture a reference to `diveLogViewModels` and rather than creating a view model from a data model (as you're doing in `cellForRowAt:indexPath`, let's use the view model for a given row. That way you don't have to create the view model as the tableView cells are rendering (speeding up performance).